configtx/orderer.go

```diff
@@ -517,6 +479,11 @@ func newOrdererGroup(orderer Orderer) (*cb.ConfigGroup, error) {
 
 	// add orderer groups
 	for _, org := range orderer.Organizations {
+		// As of fabric v1.4 we expect new system channels to contain orderer endpoints at the org level
```

Comment:
Kept this logic at the `newOrdererGroup` level instead of the `newOrdererOrgGroup` level because we reuse `newOrdererOrgGroup` when adding orderer orgs to an existing config and I believe we still want to support the ability to add orderer orgs without specifying orderer endpoints in case the old config still has a top level orderer addresses value.